app/actors/curation_concerns/actors/add_to_work_actor.rb

```diff
@@ -0,0 +1,35 @@
+module CurationConcerns
+  module Actors
+    class AddToWorkActor < AbstractActor
+      def create(attributes)
+        work_ids = attributes.delete(:in_works_ids)
+        next_actor.create(attributes) && add_to_works(work_ids)
+      end
+
+      def update(attributes)
+        work_ids = attributes.delete(:in_works_ids)
+        add_to_works(work_ids) && next_actor.update(attributes)
+      end
+
+      private
+
+        def add_to_works(new_work_ids)
+          return true unless new_work_ids.present?
```

Comment:
I discovered that when we try to remove all work_ids from an existing work, this line prevents the update from functioning. We'd like to be able to "remove all parent works from a child work" by posting an empty array of :in_works_ids .. Should this line be removed entirely, or does it serve a particular purpose?